lib/tsc.js

```diff
@@ -1,14 +1,14 @@
 /*! *****************************************************************************
-Copyright (c) Microsoft Corporation. All rights reserved. 

```

Comment:
@mhegazy :

Out of curiosity: Shouldn't the `lib` folder be omitted from being maintained by git anyway? Wouldn't it be reasonable to add `lib` to `.gitignore` and have the Travis CI generate npm/NuGet packages from its local build?